plugins/storage/entities/entities/nut_storage/init.lua

```diff
@@ -14,8 +14,6 @@ function ENT:Initialize()
 	self:SetSolid(SOLID_VPHYSICS)
 	self:SetUseType(SIMPLE_USE)
 	self.receivers = {}
-	
-	self:PostInitialize()
```

Comment:
Maybe an `isfunction(self.PostInitialize)` check could be added here in case some old code happens to expect `PostInitialize` to run.